### DISCUSSION TITLE
How to use it on multiple devices?

### DISCUSSION THREAD

**Discussion by gerroon** [cheshire-cat-ai/core#950] [Q&A] [answered]
Hi

When I have the account open it multiple tabs, browsers or devices the pages constantly refresh making it impossible  to use. It seems like I can only have it open only once on one session. Is there a way to use it on multiple devices? I want to use it on my desktop and phone at the same time without worrying about closing the other. I am trying to see if I can just use a single account as well

thanks

**Comment by pieroit** [comment]
Log in with different users

**Reply by gerroon** [reply]
Thanks but I already mentioned that I would like to avoid it " I am trying to see if I can just use a single account as well" Every device/tab will need its own username with each needing customizations etc. 

It sounds like this is not possible to achieve with single user.

**Reply by pieroit** [reply]
Hi @gerroon the problem is having multiple websocket connections open with the same user from different devices.
You can for sure use the same account from different devices, but not having  the double websocket connection

**Reply by gerroon** [reply]
I see, is there a way to use the app without the websockets?

**Reply by pieroit** [reply, marked as answer]
Yes you can build your own client using only http
